app/src/main/java/com/xmartlabs/scasas/doapp/ui/SignInFragment.java

```diff
@@ -43,10 +43,15 @@ protected int getLayoutResId() {
     return R.layout.singin_fragment;
   }
 
+  @SuppressWarnings("deprecation")
   @OnClick(R.id.sign_up)
   void onClickedSignUp() {
     Intent intent = Henson.with(getContext()).gotoSignUpActivity().build();
     getContext().startActivity(intent);
+    usernameView.setText(null);
```

Comment:
it's strange, you leave the activity and then you change the color and the text.. Why?